go/libraries/doltcore/env/environment.go

```diff
@@ -155,6 +162,50 @@ func (dEnv *DoltEnv) HasLocalConfig() bool {
 	return ok
 }
 
+// GetLicenseFile returns the path to the LICENSE.md file if it exists
+func (dEnv *DoltEnv) GetLicenseFile() string {
```

Comment:
might want to change to something like GetDoc which takes some parameter which tells it which doc.  Then if you add docs you don't have to duplicate all these functions again.

Reply:
Done: https://github.com/liquidata-inc/dolt/pull/222/files#diff-bd6777a87b5edb64be7736c8af71bbc1R165